fulp_modules/features/antagonists/bloodsuckers/code/clans/_clan.dm

```diff
@@ -225,18 +229,32 @@
 	INVOKE_ASYNC(src, PROC_REF(interact_with_vassal), bloodsuckerdatum, vassaldatum)
 
 /datum/bloodsucker_clan/proc/interact_with_vassal(datum/antagonist/bloodsucker/source, datum/antagonist/vassal/vassaldatum)
+	var/datum/mind/vassal_mind = vassaldatum.owner
+	var/datum/objective/source_objective = source.my_clan?.clan_objective
```

Comment:
Discordant Vassals mostly just exist as a way for Brujah Bloodsuckers to subvert the station's leadership. Ideally they should always be a member of command or a security officer (their objective target selection code attempts to ensure this.)

Currently the 'interact_with_vassal()' code attempts to ensure that Brujah Bloodsuckers always turn their target into a Discordant Vassal whenever they try to turn them into a special vassal type. The only alternative is allowing them to softlock themselves since special vassal types can't be removed.

Perhaps I should just make it so that their target becomes a Discordant Vassal immediately upon being vassalized. Your thoughts?